### ISSUE TITLE
ForEach with multiple variables error

### ISSUE THREAD

**Issue by coder-arn** [fastbuild/fastbuild#772]
This is with FBuild v1.01.

```
.testA = {'a', 'b'}
.testB = {'1', '2'}

ForEach( .A in .testA, .B in .testB )
{
	Print( '$A$-$B$' )
}
```

Which generates this output:

```
C:\temp\FastBUILD\fbuild.bff(4,22): FASTBuild Error #1200 - ForEach() - Expected a variable at this location.
ForEach( .A in .testA, .B in .testB )
                     ^
                     \--here
```

It seems like the comma isn't handled correctly anymore as this same script works fine in FastBuild v0.97 and generates this output:

```
a-1
b-2
Unknown build target 'all'
Time: 0.001s
```

**Comment by ffulin** [comment]
That does seem like a regression. If it's what I think it is, it'll be very easy to fix.
If it is what I thing it is, removing the comma should be an effective work-around.

**Comment by ffulin** [comment]
Thankyou for reporting the bug :)

**Comment by coder-arn** [comment]
Thank you for the quick reply!

I did look at the source which gave me the idea to remove the comma too.  It does work removing the comma but I didn't want to confuse the repro by posting that here as well =)

Unfortunately our build script errors in a different place with the comma removed.  However, I'm not sure the if the two problems are related.  I'll see if I can find a minimal repro for our second issue after the comma is removed and post it here if it's related or create a another issue if it isn't.

Thanks again!

**Comment by ffulin** [comment]
I've fixed the issues with commas not being parsed properly (https://github.com/fastbuild/fastbuild/commit/1c123b62f749db9f149adcb9782522287eabb199). This fix will be in the next release (v1.02)

I'll leave this issue open until that version is released.

Let me know when you get some more information about your other issue and we can track it here if it's related or make a new issue as appropriate.

**Comment by ffulin** [comment]
v1.02 has now been released which includes a fix for this issue. Thanks for reporting the bug!
